README.md

```diff
@@ -42,6 +68,30 @@ type Action =
   | { type: CLEAR_TODOS; payload: {} };
```

Comment:
This is a preexisting condition, but I just realized the literals `ADD_TODO`, etc. aren't quoted here and they should be.